### ISSUE TITLE
Compressor binary sensor always off

### ISSUE THREAD

**Issue by thorsten** [pail23/stiebel_eltron_isg_component#59]
First of all, pretty cool integration! I own a Stiebel Eltron LWZ 304 Trend and a ISG Plus.

## Version of the custom_component
v20235.2

## Describe the bug
The entity "binary_sensor.stiebel_eltron_isg_compressor" is always "off", although my own modbus configuration fetches the correct information.

## My working Modbus configuration
```
      - name: "LWZ BETRIEBSSTATUS"
        unique_id: "lwz_betriebsstatus"
        address: 2000
        slave: 1
        count: 1
        input_type: input
        data_type: uint16
```

and

```
  - binary_sensor:
      - name: "LWZ Kompressor aktiv"
        state: "{{ states('sensor.lwz_betriebsstatus')|int|bitwise_and(2) > 0 }}"
```

**Comment by northalpha** [comment]
Can confirm this for my LWZ 5s plus with ISG web, same modbus config and same template sensor like @thorsten 

![Bildschirmfoto 2023-05-16 um 10 33 29](https://github.com/pail23/stiebel_eltron_isg_component/assets/6721009/97b7f484-502a-46dd-8e29-f93bce62c8fc)

**Comment by pail23** [comment]
This should be fixed in 2023.5.3. Could you please test the new release?

**Comment by thorsten** [comment]
@pail23 yes, it's fixed with the 2023.5.3 release, thanks a lot.
